web/packages/extension/assets/_locales/he/messages.json

```diff
@@ -94,5 +94,20 @@
     },
     "settings_saved": {
         "message": "ההגדרות נשמרו"
+    },
+    "onboarding_page": {
+        "message": "דף נחיתה"
+    },
+    "permissions_grant": {
+        "message": "הענק הרשאה לRuffle לגשת לנתונים שלי בכל האתרים"
+    },
+    "permissions_not_granted": {
+        "message": "עליך ללחוץ על הסמליל של Ruffle בכל אתר בו תרצה להשתמש בRuffle"
+    },
+    "grant_single_site_permission": {
+        "message": "אפשר לתמיד באתר זה"
+    },
+    "permissions_explanation": {
+        "message": "התוסף של Ruffle עובד הכי טוב עם היכולת לגשת לנתונים שלך בכל האתרים. יכולת זו תאפשר לו לפעול ברגע שהאתר נטען, ומאילך, יעזור להתחזות לתוסף הפלאש באתרים ובכך יאפשר לתוכן הפלאש לפעול כשורה. בנוסף, זה יתן לו את היכולת לטעון קבצי SWF בתוך הנגן העצמאי שלו ממיקומים שונים של האתר. אם לא תאפשרו גישה זו, תצטרכו להפעיל את Ruffle באופן עצמאי בכל אתר שבו תרצו להפעיל תוכן פלאש בכך שתלחצו על סמליל התוסף ותאפשרו גישה לאתר שבו אתם נמצאים."
```

Comment:
Um, there might be something funky going on with this string, maybe around the direction switching characters around `SWF`? :thinking:
Pasting it into Google Translate, it gets cut off right around there...

Reply:
You sure you copied the string properly? It's RTL so selecting it is a little annoying...

Reply:
> It's RTL so selecting it is a little annoying...

I know, that's why I always select from/to LTR English parts (like JSON keys) a few lines above/below the interesting string, to be sure.

Does it work for you? I get this:
![image](https://github.com/ruffle-rs/ruffle/assets/288816/b4ed11e2-de79-4e02-801e-b6ce4bd7c523)

However, after deleting `SWF` plus a few words before/after:

![image](https://github.com/ruffle-rs/ruffle/assets/288816/f415ec38-fc5f-44d0-85bc-f110fea99f25)

Reply:
I think the double apostrophe at the end breaks Google Translate... If I copy the translated string 1:1 it's okay

Reply:
Huh... Weird. But okay then!